packages/babel-traverse/src/visitors.ts

```diff
@@ -222,9 +224,43 @@ export function merge(
     for (const type of Object.keys(visitor) as (keyof Visitor)[]) {
```

Comment:
If `visitor` has `enter` or `exit`, then it is a catch all visitor. In this case we don't have to go through its properties, instead we can call `wrapWithStateOrWrapper` on `.enter` and `.exit` and push it to the rootVisitor. In this approach we don't have to handle `_exploded` and `_verified`.